plugins/CoreHome/stylesheets/layout.less

```diff
@@ -195,7 +205,7 @@ nav {
 
   #secondNavBar {
     width: 224px;
-    float:left;
+    height: fit-content;
```

Comment:
If we add `flex: none` here, we don't need to do any width setting on the `.pageWrap` at all (the calc, the 100% on mobile..)

```suggestion
    height: fit-content;
    flex: none;
```

Reply:
This doesn't have the expected effect. I will add it, as it ensure the menu won't be resized, but the width on the pagewrap os still needed. The problem is, that by default the pagewrap would only use the minimal required space. But it actually should take all space that is available. Otherwise the reports might not be as wide as they were before.
You can see that when removing the width properties and viewing e.g. a pages report. The report would then not stretch across the whole screen anymore

Reply:
I'll recheck what I'd disabled when I was testing the PR in a browser.. might have been removing the calc but keeping 100% everywhere (definitely removing the `calc`).

Reply:
Only using 100% might now also work. The calc was needed, as 100% tried to consume too much space. But with flex: none, that might work now. I'll give it a try.

Reply:
@sgiehl I went over the majority of screenshots in the 'with 100% width' version andthe changes around notifications are actually improvements over what it is now.
Since we're changing so many screenshots already, I'd say the goal doesn't need to be necessarily to match all one for one, but small improvements in the CSS (like we have here) resulting in small improvements in the UI (like we may have here) should be fine.

Basically the main difference is that the notification behaves better with 100% width compared to the calc version (see screenshots below, don't mind the temporary red background used for easier visualisation).

With calc (as we have now - notice the slight indent inward)
![notification with calc](https://github.com/matomo-org/matomo/assets/233342/e180221d-6a26-4353-bc63-5e2b5382bdb4)


With 100% width (better aligned, no indent), not functional change in the actual UI in browser.
![notification with 100 width](https://github.com/matomo-org/matomo/assets/233342/df6e0ae2-91fb-4d56-83f7-da24466aecab)

I'd say it'd be better to have flex:none and width: 100% for the rest and update the additional UI tests as they better reflect the improvement, but I'd leave the final decision on this to you.

Reply:
Thought the same when looking at those. But there are also changes like:
https://builds-artifacts.matomo.org/matomo-org/matomo/menushadow/7248776660/UIIntegrationTest_actions_pages.png
https://builds-artifacts.matomo.org/matomo-org/matomo/menushadow/7248776660/GoalsPages_overview.png
https://builds-artifacts.matomo.org/matomo-org/matomo/menushadow/7248776660/PivotByDimension_pivoted.png
Those changes look unexpected and would need further investigation

Reply:
Yeah, I agree, some of the tests may need to looked at manually in the UI. The line-height change doesn't make much sense to me, so might be a one off maybe, re-running the tests could help?
I can look into those in a PR on top of this PR. I guess we can iterate either here or in a separate PR until we decide to update UI tests in plugins and until we decide it's too much time for too little benefit.

Reply:
I gave removing the calculated width another try. The main problem is the reports by dimensions view for goals and referrers. As they add another floating, that breaks the flex layout, or at least the width. The element itself takes 100% space (ignoring the menu) and is getting too big then. I tried fixing it, by using flex there as well, but did find a quick fix for it. Imho we should go with this one as it is for now. Guess we would need to fully rework some parts of the layout to make that work correctly. And that's not really worth the effort only to get rid of a calculated width...

Reply:
Thanks for looking into this again. I agree, we will hopefully get to addressing the UI from a bigger picture perspective at some point soon, so this is good for now.